Add Invisible Institute ruleset

VoidPhantom opened the pull request.

gloomy-ghost commented:
Please add targets:
the.invisible.institute
www.the.invisible.institute

Please note as `Invalid certificate` in a top comment:
cpdb.invisible.institute
data.invisible.institute

gloomy-ghost commented on src/chrome/content/rules/Invisible.institute.xml:
```diff
@@ -0,0 +1,8 @@
+
+	<rule from="^http:" to="https:" />
+
+	<securecookie host="^invisible\.institute$" name=".+" />
```
`__utma` is under `.invisible.institute`

jeremyn commented:
@VoidPhantom Are you still interested in working on this?

VoidPhantom commented:
Sorry for the long absence. :frowning_face:

I added the.invisible.institute. I also found and added securedrop.invisible.institute (this subdomain doesn’t seem to have anything important, but it’s there). www.the.invisible.institute, cpdb.invisible.institute, and data.invisible.institute don’t seem to exist (anymore).

jeremyn commented:
Please:

- [ ] Change the `ruleset` `name` to `Invisible.institute`

jeremyn commented:
Thanks, merged.
